src/curtis/__main__.py

```diff
@@ -95,13 +112,31 @@ def browse_unseen_issues(cfg, age):
     help="Issues with a trend ratio below this threshold are ignored",
     show_default=True,
 )
+@click.option(
+    '--filter',
+    '-f',
+    multiple=True,
+    help="Patterns of issue titles to filter out (Python regular expression)",
+)
+@click.option(
+    '--exclude',
+    '-e',
+    multiple=True,
+    help="Patterns of issue titles to exclude (Python regular expression)",
+)
 @click.pass_obj
-def check_trends(cfg, period, threshold):
+def check_trends(cfg, period, threshold, filter, exclude):
     """Show evolution stats for seen issues"""
     stats_period, days, hours = utils.decode_period(period)
 
     issues = client.opi_iterator(cfg, query='is:unresolved is:unassigned', statsPeriod=stats_period)
     issues = filters.max_age(filters.seen(issues), days=days, hours=hours)
+
+    for exp in filter:
+        issues = filters.filter_title(issues, regex=re.compile(r'^.*%s.*$' % exp))
```

Comment:
`r'^.*%s.*$' % exp` could be replaced by `exp`, no ?

Reply:
I thinks that this could be a good use case for https://docs.python.org/3.6/library/functools.html?highlight=reduce#functools.reduce